1225-my_institution/institution_administrator_edits_my_institution.feature

```diff
@@ -21,5 +22,19 @@ Feature: Institution Administrator edits My Institution
             | Display name       |
             | Short display name |
             | Archive name       |
+            | Vocabularies       |
         And they click Save
         Then they see a Notification that the changes were saved
+
+    @xxx
+    Scenario Outline: Institution Administrator see fields for Vocabularies
+        Given that the user is logged in as Institution Administrator
+        And they click the menu item My Institution
+        And they see the My Institution page
+        When they see fields for "<Vocabulary>"
+        Then they can toggle value for "Default"
+        And they can toggle value for "Active"
```

Comment:
jeg skjønner ikke hva som skal gjøres ut ifra tekstlig beskrivelse her. Default hva for noe? Active hva for noe?

Reply:
Det er et dårlig tegn :p 
Admin skal kunne sette disse listene (Vocabularies) til "Default" (vises i wizard som default) og "Active" (Bruker kan legge til om ønskelig).
Utkast:
![bilde](https://user-images.githubusercontent.com/6313468/111132135-4465ee80-8579-11eb-8b5b-2241c8379d85.png)
![bilde](https://user-images.githubusercontent.com/6313468/111132263-711a0600-8579-11eb-8d01-4fed4318814b.png)

Reply:
kanskje vi kan få med at dette er en innstilling som settes og at det påvirker oppslaget i NPI-feltet?

kanskje skrive et scenario for hva som skjer når en admin huker av "Active" og hva som skjer når en huker av "Default"?

Reply:
sorry, litt misvisende eksempel, for NPI-fagfelt er egentlig ikke relevant her, men hvis admin har satt en liste til "Default" vil det se ut som NPI-fagfelt her, så kan brukeren legge til andre lister som er "Active" ved å trykke linken under. 

Det andre scenarioet jeg la til beskriver at brukern legger til en "Active" liste, så tenkte jeg at "Default" bare kunne være implisitt... Skulle vi heller hatt 2 scenario der? `Creator adds a Vocabulary` (Given institution has an Active Vocabulary) og `Creator adds value for a Vocabulary` (Given institution has a Default Vocabulary or Creator has added a Active Vocabulary)?

Reply:
ok, vet ikke om jeg helt skjønte det heller 😝 
er det forskjell på funksjonalitet for disse 4 alternativene?
1. Ikke Default og Ikke Active
2. Ikke Default og Active
3. Default og Ikke Active
4. Default og Active

Isåfall må vi beskrive hva som skjer i alle casene

Reply:
kan vi i det hele tatt skru av default? rart at det er en toggle-knapp isåfall?

Reply:
ja, de skissene der er noe som er kokt samme i wireframes for å vise flyten, Kine skal se på det før det kan implementeres.

1. Ikke Default og Ikke Active -> admin har bestemt at Creators ikke skal kunne legge til ting herfra
2. Ikke Default og Active -> brukere kan velge å legge til denne manuelt
3/4. Default  -> Setting for Active har ingen betydning, siden feltet skal vises by default for alle registreringer

Kanskje like greit å sette denne til draft til Kine har sett på det, og vi vet hvordan admin skal sette disse verdiene

Reply:
men er det kanskje selve ordene `Default` og `Active` jeg har misforstått da? 
Kan det samme beskrives med `Default` = `Mandatory` og `Active` = `Optional`?
Slik at det egentlig er tre valg her? 
1. Mandatory
2. Optional
3. Off

Reply:
Det er ikke mandatory heller. Hvis det er "Default" skal Creator kunne legge til verdier til det feltet som standard (feltet vises automatisk). Hvis det er "Active" kan Creator legge til verdier der også, men man må først trykke link for velge hvilken liste man vil vise. I første runde er det bare 2 lister (Vocabularies), men på sikt blir det 10-ish(?)

Hvis det er Default betyr det at det er relevant for de _fleste_ registreringer ved den institusjonene, hvis det er Active betyr det at det kan være aktuelt for _enkelte_ registreringer

Reply:
ok, tror jeg skjønte det nå 😝 `Default` gir hvertfall mening. Men synes fortsatt at `Active` ikke er beskrivende nok.

Reply:
Enig, det er kanskje heller "Allowed" elns. men tror jeg setter dette til draft fo nå, kan være Kine finner på noe lurt når hun skal lage skisser

Reply:
Eg har ikkje lagt opp til at det skal vere mulig å påkreve bruk av eit kontrollert vokabular - det lyt ein brukar be om før det blir eit krav. Intensjonen er å la ein institusjon velge ut dei vokabular som passar for bruk ("Allowed") og sette til "Default" dei som ønskes brukt.